src/pages/[lang]/volunteers.astro

```diff
@@ -0,0 +1,395 @@
+                                                            volunteer.links
+                                                                .youtube
+                                                        }
+                                                        social="youtube"
+                                                    />
+                                                )}
+                                                {volunteer.links.git && (
+                                                    <SocialsComponent
+                                                        speakersSocialUrl={
+                                                            volunteer.links.git
+                                                        }
+                                                        social="github"
+                                                    />
+                                                )}
+                                            </div>
+                                        </div>
+                                    </div>
+                                ))}
+                            </div>)}
+                            
+                        </>
+                    ),
+                )
+            }
+        </div>
+
+
+        <script define:vars={{ volunteers }}>
+            //Define switch variable
+            let groupedSelectedBtn = true;
+            //
+
+            //Just upsdates de switch buttons styles
+            function updateSwitchButtons() {
+                const allBtn = document.getElementById("all-btn");
+                const groupedBtn = document.getElementById("grouped-btn");
+
+                if (allBtn && groupedBtn) {
+                    if (groupedSelectedBtn) {
+                        allBtn.classList.remove(
+                            "bg-gradient-to-r",
+                            "from-purple-600",
+                            "to-pink-600",
+                            "text-white",
+                        );
+                        allBtn.classList.add(
+                            "text-gray-700",
+                            "dark:text-gray-300",
+                            "hover:bg-gray-100",
+                            "dark:hover:bg-gray-700",
+                        );
+
+                        groupedBtn.classList.remove(
+                            "text-gray-700",
+                            "dark:text-gray-300",
+                            "hover:bg-gray-100",
+                            "dark:hover:bg-gray-700",
+                        );
+                        groupedBtn.classList.add(
+                            "bg-gradient-to-r",
+                            "from-purple-600",
+                            "to-pink-600",
+                            "text-white",
+                        );
+                    } else {
+                        // UTC is active
```

Comment:
This looks like a leftover from the schedule switch. Maybe some of these can be removed, after all we only are using the switch